### DISCUSSION TITLE
v0.4.37

### DISCUSSION THREAD

**Discussion by lbbrhzn** [lbbrhzn/ocpp#749] [Announcements]
## Changes

* Update requirements_test.txt (#748) @lbbrhzn
* Naming tidy ups (#676) @drc38
* Add Recommendation for Vestel EVC-04 Wallboxes (#739) @N3rdix
* Add link to getting started guide for CTEK (#744) @moorgrove
* Update .pre-commit-config.yaml (#747) @lbbrhzn
* Add United Chargers Grizzl-E to documentation (#675) @BjarneBitscrambler

## :package: Dependencies

* build(deps): bump actions/setup-python from 4.3.0 to 4.5.0 (#737) @dependabot
* build(deps): bump actions/upload-artifact from 3.1.1 to 3.1.2 (#732) @dependabot
* build(deps): bump release-drafter/release-drafter from 5.21.1 to 5.22.0 (#728) @dependabot
* build(deps): bump isort from 5.10.1 to 5.11.4 in /.github/workflows (#721) @dependabot
* build(deps): bump black from 22.10.0 to 22.12.0 in /.github/workflows (#708) @dependabot
* build(deps): bump ocpp from 0.15.0 to 0.16.0 (#731) @dependabot
* build(deps): bump pre-commit from 2.20.0 to 2.21.0 in /.github/workflows (#723) @dependabot
* build(deps): bump actions/stale from 6 to 7 (#719) @dependabot
* build(deps): bump pyupgrade from 3.2.0 to 3.3.1 in /.github/workflows (#704) @dependabot
* build(deps): bump flake8 from 5.0.4 to 6.0.0 in /.github/workflows (#692) @dependabot

<hr /><em>This discussion was created from the release <a href='https://github.com/lbbrhzn/ocpp/releases/tag/v0.4.37'>v0.4.37</a>.</em>

**Comment by kwanm84** [comment]
Could not install 0.4.37 on Home Assistant 2022.11.2 (I know I need to update home assistant, it was easier to install a previous version of ocpp than update as updates normally break something on my home assistant installs)
Had a flow config error.

I installed 0.4.36 without issue.

**Reply by N3rdix** [reply]
I had no issues with 0.4.37 on 2022.12 when I updated. Sounds more like an issue with your setup.
Still runs fine with the latest HA release
